recipes/grass-session/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "grass-session" %}
+{% set version = "0.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 7155314535790145da8e2e31b0d20cd2be91477d54083a738b5c319164e7f03b
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Comment:
I think this package can probably be `noarch: python`

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python